app/embeds/GoogleMaps.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
We moved over to Typescript quite a while ago now, all types should be Typescript and all file extensions should be `.ts` – see all the other extensions in the codebase.